tutorials/tutorial-howto-monitor-django-prometheus.md

```diff
@@ -0,0 +1,230 @@
+# Tutorial: How to monitor a Django application with Prometheus
+
+>:TOPLIST:
```

Comment:
Using a `TOPLIST` requires an H3 at the beginning of the list
```suggestion
>:TOPLIST:
> ### Sections
```
or similar

Reply:
Having a TOPLIST has not been used with other tutorials, so if we are going to start doing this, we need to add it to all tutorials.

Reply:
I don't see much harm in testing this structure with a few tutorials first? I don't think any user is going to fault us too much for having a table of contents sidebar in some tutorials but not others.